### DISCUSSION TITLE
We've encountered some adaptation issues with the new fastdds （V2.10.3 or master）version on Apollo. Has anyone performed a migration before? I'd like to seek advice on this matter.

### DISCUSSION THREAD

**Discussion by lobmoo** [ApolloAuto/apollo#15396] [General]
Currently, after the migration, shared memory (shm) communication is working fine, but there's a deadlock occurring with global RTPS (Real-Time Publish-Subscribe) operations.

**Comment by daohu527** [comment]
Can you describe the problem in detail, there is a path for fastrtps `docker/build/installers/FastRTPS_1.5.0.patch`.

You can try modifying the patch to see if it has any impact.

**Reply by lobmoo** [reply]
能留个联系方式交流一下嘛

**Reply by daohu527** [reply]
email: daohu527@gmail.com
